stellargraph/layer/gcn_lstm.py

```diff
@@ -212,41 +195,41 @@ def call(self, features):
 class GraphConvolutionLSTM:
 
     """
-        A stack of 2 Graph Convolutional layers followed by an LSTM, Dropout and,  Dense layer.
+        A stack of N1 Graph Convolutional layers followed by N2 LSTM, Dropout and,  Dense layer.
         This main components of GNN architecture is inspired by: T-GCN: A Temporal Graph Convolutional Network for Traffic Prediction
                                           (https://arxiv.org/abs/1811.05320)
         The implementation of the above paper is based on one graph convolution layer stacked with a GRU layer.
-        The StellarGraph implementation is built as a stack of the following layers:
-           1. 2-layer graph convolutional network (gcn) proposed by Kipf & Welling in  https://arxiv.org/abs/1609.02907 (ICLR 2017).
-           2. 1 LSTM layer
+        The StellarGraph implementation is built as a stack of the following set of layers:
+           1. User specified no. of Graph Convolutional layers
+           2. User specified no. of LSTM layers
            3. 1 Dense layer
            4. 1 Dropout layer
            The last two layers consistently showed better performance and regularization experimentally.
```

Comment:
I think the docstring has changed a fair bit now, do you better performance compared to the implementation in the paper?

Reply:
Actually its very hard to compare with the paper. The paper reports the results different from what I get when I run their code. And their implementation is quite different from what they propose in the paper :-). 
But this implementation does better than the simple LSTM or Naive baseline.